tests/integration/pilot/revisioncmd/revision_view_test.go

```diff
@@ -100,10 +99,6 @@ func TestRevisionCommand(t *testing.T) {
 			}
 			builder.BuildOrFail(t)
 
-			// Wait some time for things to settle down. This is very
-			// important for gateway pod related tests.
-			time.Sleep(30 * time.Second)
```

Comment:
I see this sleep command has been here for the past two years - was there a related change that eliminates the need for this sleep?  Revision code/logic stabilize enough?

Reply:
I believe it was just a poor use of sleep - I had plannedto refactor to remove it but didn't see anywhere that really needed it. Ill run the tests a few times to verify though